app/components/main-content/PrelaunchSignUpForm.tsx

```diff
@@ -90,18 +87,15 @@ const PrelaunchSignUpForm: React.FC<PrelaunchSignUpFormProps> = ({ setFlowState
 
           <section aria-label="Sign Up with Email">
             <h3>or sign up with email</h3>
-            <TextInput
-              required
-              label="Name"
-              {...form.getInputProps('name')}
-            />
-
-            <TextInput
-              required
-              label="Email"
-              {...form.getInputProps('email')}
-            />
-
```

Comment:
Only using `<TextInput/>` for all input fields at the current state. May incorporate `<Select/>` for certain input fields (there is another Jira ticket for this purpose) when we come up with comprehensive dropdown options.